### PR TITLE
Update Request Matching.md

### PR DESCRIPTION
**Pull request by Smartmind12**
Adding codetabs as part of issue 183.

## References

- Part of #183 

## Submitter checklist

- [X] The PR request is well described and justified, including the body and the references
- [ ] The PR title represents the desired changelog entry
- [ ] If the change against WireMock 2 functionality (incompatible with WireMock 3),
      it is submitted against the [2.x](https://github.com/wiremock/wiremock.org/tree/2.x) branch
- [ ] The repository's code style is followed (see the contributing guide)

_Details: [Contributor Guide](https://github.com/wiremock/wiremock.org/blob/main/CONTRIBUTING.md)_

### CODE REVIEW

**Review by oleg-nenashev — COMMENTED** [review]
```
  Liquid Exception: Liquid syntax error (line 645): Unknown tag 'endcodetab' in /home/runner/work/wiremock.org/wiremock.org/_docs/request-matching.md
                    ------------------------------------------------
      Jekyll 4.3.2   Please append `--trace` to the `build` command 
                     for any additional information or backtrace. 
                    ------------------------------------------------
```

File: _docs/request-matching.md [hunk 1]
```diff
@@ -500,6 +631,10 @@ JSON:
 }
 ```
 
+{% endcodetab %}
+
+{% endcodetabs %}
```
Comment (oleg-nenashev) [inline _docs/request-matching.md:636]: Mistake here

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
Requesting changes based on the comment above

File: _docs/request-matching.md (hunk 1 above)
Comment (Smartmind12) [reply _docs/request-matching.md:636]: Yes I have editted it!

**Comment by tomakehurst** [comment]
Closing this as we're migrating the docs to mkdocs with a new tabbed code feature.

**Pull request closed without merging**